front/components/assistant/conversation/AgentSuggestion.tsx

```diff
@@ -22,6 +22,7 @@ export function AgentSuggestion({
   userMessage: UserMessageType;
   conversation: ConversationType;
 }) {
+  const editingMessage = useRef(0);
```

Comment:
Should we do `isEditingMessage` state which is more aligned with what we do eslewhere?

Reply:
`isEditingMessage` naming is definitely better. But by state you meant a ref (from `useRef`) right?

Reply:
I was more thinking like using a `useState` and prevent the click on the button which is what we do elsewhere. Less direct but equally effective no?

Reply:
as discussed on Slack, we are moving forward with useRef().